packages/ui/packages/Carousel/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+// import { Slider } from 'es6-simple-carousel';
+import { Slider } from './src/index';
+import { StyledSimpleCarousel } from './src/styles';
+
+function SimpleCarousel(props) {
```

Comment:
it is better to use `const fnName = (props) => `

Reply:
done